### PR TITLE
Add instructions for making a release build

### PR DESCRIPTION
**Pull request by mvd-ows**
As above.

---
This change is [Reviewable](https://reviewable.io/reviews/mullvad/win-split-tunnel/8)

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Making a release build
+
+## One-time setup
+
+These steps are performed on a trusted build machine.
+
+Ensure the following software is installed:
+
+    1. `Visual Studio Community 2019` or later.
+    1. `Windows Driver Kit (WDK)` version `10.0.18362.1` or later.
```
Comment (faern) [inline RELEASE.md:10]: I don't think these bullet lists are rendered the way you intended them to:

https://github.com/mullvad/win-split-tunnel/blob/add-release-instructions/RELEASE.md

**Review by mvd-ows — COMMENTED** [review]
Thanks, fixed.

> *[Reviewable](https://reviewable.io/reviews/mullvad/win-split-tunnel/8#-:-MVIQstBDGLpxdryi6RS:b-x52gbo)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @dlon and @faern)

**Review by dlon — APPROVED** [review]
> Reviewed 1 of 1 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/mullvad/win-split-tunnel/8)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @faern)

___
*[RELEASE.md, line 9 at r1](https://reviewable.io/reviews/mullvad/win-split-tunnel/8#-MVIarCc6So_tR1FAYWz:-MVIarCd86AfxKebm0mr:bkarzf4) ([raw file](https://github.com/mullvad/win-split-tunnel/blob/49709f30a9a8f2d3b2ef4a0450848210a25062b5/RELEASE.md#L9)):*
> ```Markdown
> Ensure the following software is installed:
> 
> 1. `Visual Studio Community 2019` or later.
> ```

I think any edition would work fine.

**Review by mvd-ows — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/win-split-tunnel/8)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @faern)

___
*[RELEASE.md, line 9 at r1](https://reviewable.io/reviews/mullvad/win-split-tunnel/8#-MVIarCc6So_tR1FAYWz:-MVIeB606D8RkaKO9V-a:b-x9aaxy) ([raw file](https://github.com/mullvad/win-split-tunnel/blob/49709f30a9a8f2d3b2ef4a0450848210a25062b5/RELEASE.md#L9)):*
<details><summary><i>Previously, dlon (David Lönnhager) wrote…</i></summary><blockquote>

I think any edition would work fine.
</blockquote></details>

Agree. Do you think it should be more generally worded?

**Review by faern — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/win-split-tunnel/8)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mvd-ows)

___
*[RELEASE.md, line 37 at r2](https://reviewable.io/reviews/mullvad/win-split-tunnel/8#-MVIw-Yu-gBr1lNEuM3F:-MVIw-Yu-gBr1lNEuM3G:bj5tyym) ([raw file](https://github.com/mullvad/win-split-tunnel/blob/53166937b0aea0010cec2aaf320d1a4155b1a8ff/RELEASE.md#L37)):*
> ```Markdown
> 1. Artifacts are prepared under `/bin/dist`:
>     1. `/bin/dist/legacy` final artifacts for Windows 7/8/8.1.
>     1. `/bin/dist/win10` intermediate driver package for Windows 10.
> ```

Nit. From a unix point of view all of these paths look like absolute paths. I would have written them as `bin/dist` etc.. But if this is some Windows convention I'm fine with it.

**Review by faern — APPROVED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/mullvad/win-split-tunnel/8)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mvd-ows)

**Review by mvd-ows — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/win-split-tunnel/8)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mvd-ows)

___
*[RELEASE.md, line 37 at r2](https://reviewable.io/reviews/mullvad/win-split-tunnel/8#-MVIw-Yu-gBr1lNEuM3F:-MVKvYG92QEP9YWmuMIj:b-shk57l) ([raw file](https://github.com/mullvad/win-split-tunnel/blob/53166937b0aea0010cec2aaf320d1a4155b1a8ff/RELEASE.md#L37)):*
<details><summary><i>Previously, faern (Linus Färnstrand) wrote…</i></summary><blockquote>

Nit. From a unix point of view all of these paths look like absolute paths. I would have written them as `bin/dist` etc.. But if this is some Windows convention I'm fine with it.
</blockquote></details>

In a haze, the leading slash was included to indicate these paths are rooted in the repository, as opposed to being relative within the repository. I'll correct this shortly.

**Review by dlon — APPROVED** [review]
> Reviewed 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/mullvad/win-split-tunnel/8)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mvd-ows)

**Pull request merged**